bin/compile

```diff
@@ -22,3 +23,12 @@ echo "exporting PATH" | indent
 PROFILE_PATH="$BUILD_DIR/.profile.d/ffmpeg.sh"
 mkdir -p $(dirname $PROFILE_PATH)
 echo 'export PATH="$PATH:${HOME}/vendor/ffmpeg"' >> $PROFILE_PATH
+export PATH="$PATH:$BUILD_DIR/vendor/ffmpeg"
```

Comment:
Pourquoi est-ce nécessaire ?

Reply:
Sinon le build fait un ffmpeg command not found et du coup exit 1

Reply:
OK. Par ailleurs, un échec du script fait bien échouer le build ?

Reply:
Oui, ça fait bien échouer le build